app/views/documents/_filter_table.html.erb

```diff
@@ -0,0 +1,61 @@
+<% if filter[:results].any? %>
```

Comment:
Looks good! Something like this would be a prime candidate to unit test if we every decide to use something like view components though! There's a lot going on

Reply:
Yup, it was quite a delicate operation to check it was all outputting correctly.